### PR TITLE
Standardize docker-compose and integrate with .env variables

### PR DESCRIPTION
**Pull request by miguelpeixe**
A lot of the variables already declared in .env were not being used to build the docker instance, resulting in database and minio conflicts.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -147,26 +147,28 @@ services:
...
   # https://docs.min.io/docs/minio-client-complete-guide#policy
   minio_client:
     image: minio/mc
     depends_on:
       - minio
+    environment:
+      - S3_REGION
     entrypoint: >
       /bin/sh -c "
       /usr/bin/mc config host add coopcycle http://minio:9000 minio coopcycle;
```
Comment (miguelpeixe) [inline docker-compose.yml:164]: Not sure how to build this config line, help needed

File: docker-compose.yml [hunk 2]
```diff
@@ -86,10 +86,10 @@ services:
       - php
     restart: always
     environment:
-      SYMFONY_ENV: dev
-      GOOGLE_API_KEY: ${GOOGLE_API_KEY}
-      STRIPE_PUBLISHABLE_KEY: ${STRIPE_PUBLISHABLE_KEY}
-      STRIPE_SECRET_KEY: ${STRIPE_SECRET_KEY}
+      - SYMFONY_ENV=${APP_ENV}
```
Comment (alexsegura) [inline docker-compose.yml:89]: `SYMFONY_ENV` is not used anymore it's legacy it can be removed

File: docker-compose.yml (hunk 1 above)
Comment (miguelpeixe) [reply docker-compose.yml]: Just sent an update, [minio docs](https://docs.min.io/docs/minio-client-complete-guide.html) helped me out

**Comment by miguelpeixe** [comment]
@alexsegura is there anything else needed for this PR to be merged?

**Comment by alexsegura** [comment]
Thank you @miguelpeixe, sorry for taking so long to merge

**Comment by alexsegura** [comment]
@miguelpeixe actually, there was a small thing missing 😉 

1ab7bec2af163a9008aaac48989a907fd0283e09

**Comment by miguelpeixe** [comment]
Great! Thanks @alexsegura ;)

**Pull request merged**